### ISSUE TITLE
Only GAMES folder being scanned by WebMAN-MOD over network

### ISSUE THREAD

**Issue by Zoomer1988** [aldostools/webMAN-MOD#328]
I can't seem to get WebMAN-MOD recognize any other folder, namely PS3ISO, PSXISO etc, except GAMES over network. 

I've setup latest PS3NETSRV on Windows PC and pointed it to a folder which contains all other subfolders mentioned above. Everything gets scanned in GAMES folder and works alright. Yet all other folders are being ignored by scanning routine. I have a PS3 BD iso of Afrika in PS3ISO folder and couple of PSX games in PSXISO foder in bin/cue format. 

I still can get into those folders through WMM's file explorer and mount an image from there. Could someone help me please with that issue? I can't find a mention of that anywhere. Everyone seems happy with current builds. 

My version WMM version is 1.47.25.8 Full. The CFW is Rebug 4.85.1 Lite. PS3 connected to the local network via ethernet cable.

**Comment by J013k** [comment]
Did you tried webMAN MOD 1.47.26:
[https://github.com/aldostools/webMAN-MOD/releases](https://github.com/aldostools/webMAN-MOD/releases).

Also you can try:
webMAN Games -> webMAN Setup -> Refresh My Games -> Reboot PS3.
webMAN Games -> webMAN Setup -> Refresh Web Games List -> Reboot PS3.

**Comment by Zoomer1988** [comment]
Yeah, I've just tried latest release - the issue is still there. Tried both refresh options, it's all the same. Could anyone please confirm that 1.47.25.8 and later load PSXISOs over network just fine? Or DVDISOs or PS3ISOs for example.

Seems there's definitely an issue if you try serving multiple content types over network. Check out this thread on psx-place forum: https://www.psx-place.com/threads/webman-mod-ignoring-isos.27262/

As the author suggests, I've tried running a second server with only PSXISO folder populated. Now PSX games appeared in the "NET" tab together with folder games from the first running server. However, I can't get XML parser to recognize them still. Also, running two servers is not a good workaround for me since usually I'm using SynoCommunity package by anonym-tsk for my Synology NAS and I can run only one instance of it (except before opening up this ticket here I've setup PS3NETSRV under Windows, so that's not the problem with Synocommunity package).

Could someone please confirm or disprove the issue being described here? Before I try resetting my system and trying from scratch (don't want to mess around with my savegames).

**Comment by Crawdadius** [comment]
Hey, just wanted to mention that I experienced this issue as well while using webMAN-MOD 1.47.27 with ps3netsrv build 20190630.  After upgrading to the latest build of ps3netsrv (20200520) the issue was resolved and scanning of all folders works as expected.

**Comment by IzaacJ** [comment]
This issue is present in the latest ps3netsrv (20220813), it is not listing my games in PS3ISO, but they show up and run properly using 20220421 or 20220330.

Is there any reason I shouldn't go back to an older version of ps3netsrv?

**Comment by aldostools** [comment]
If the build 20220421 is working better on your environment, you should use that build.

The latest build 20220813 basically changes the execution permissions on linux according to this pull request:
https://github.com/aldostools/webMAN-MOD/pull/762
